app/commands/bootcamp/chat_message/trigger_llm.rb

```diff
@@ -1,7 +1,6 @@
 class Bootcamp::ChatMessage
-  class TriggerLLM
+  class TriggerLlm
```

Comment:
This was right before. You just needed to restart your server. Can you revert pls?